models/post_like.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require("mongoose");
+
+const PostLikeSchema = new mongoose.Schema({
+  post: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "Post",
+    required: true,
+  },
+  user: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "User",
+    required: true,
+  },
+  createdAt: {
+    type: Date,
+    default: Date.now(),
+  },
```

Comment:
Don't know if this would be needed for likes?

Reply:
You are right. They wouldn't be required. My bad.